### ISSUE TITLE
We have no twistedchecker CI

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8906]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)| @glyph reported|
|-|-|
|Trac ID|trac#8906|
|Type|task|
|Created|2016-11-18 03:06:07Z|
|Branch|https://github.com/twisted/twisted/tree/8906-twistedchecker|

We turned off buildbot, and the txchecker build has been both (A) allow_fail since it was introduced and (B) a total no-op since the `src/` move.

<details><summary>Searchable metadata</summary>

```
trac-id__8906 8906
type__task task
reporter__glyph glyph
priority__normal normal
milestone__None None
branch__8906_twistedchecker 8906-twistedchecker
branch_author__ 
status__closed closed
resolution__fixed fixed
component__release_management release management
keywords__None None
time__1479438367711703 1479438367711703
changetime__1479447321173401 1479447321173401
version__None None
owner__glyph glyph
cc__radix
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)<a name="note_3"></a>|@glyph commented|
|-|-|

Without the use of the `usedevelop` flag, the tox environment is pointed at code that `diff-quality` can't identify as in the repository.

This change fixes it so that the txchecker build actually does something, and also that it's not allowed to fail while giving a green status; we should really start enforcing the coding standard again, I think a lot of code that violates it has gotten in while we haven't been paying attention.

**Comment by twisted-trac** [comment]
|[<img alt="moshez's avatar" src="https://avatars.githubusercontent.com/u/422703?s=50" width="50" height="50">](https://github.com/moshez)<a name="note_4"></a>|@moshez set owner to @glyph|
|-|-|

reviewed on gh

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)<a name="note_5"></a>|@glyph set status to `closed`|
|-|-|

In changeset dc0f651e481e22daab8cec8b8126d958cd309f6d
```
#!CommitTicketReference repository="" revision="dc0f651e481e22daab8cec8b8126d958cd309f6d"
Merge 8906-twistedchecker: turn on twistedchecker

Author: glyph

Reviewer: moshez

Fixes: #8906

Without the use of the usedevelop flag, the tox environment is pointed
at code that diff-quality can't identify as in the repository.

This change fixes it so that the txchecker build actually does
something, and also that it's not allowed to fail while giving a green
status; we should really start enforcing the coding standard again, I
think a lot of code that violates it has gotten in while we haven't been
paying attention.
```
